test/spec/modules/sortableAnalyticsAdapter_spec.js

```diff
@@ -203,11 +202,11 @@ describe('Sortable Analytics Adapter', function() {
         brc: 1,
         brid: ['10141593b1d84a', '37a8760be6db23'],
         rs: ['300x250', '728x90'],
-        btcp: [0.70, 0.50],
+        btcp: [0.70 * 0.95, 0.50 * 0.95],
         btcc: 'USD',
         btin: true,
         btsrc: 'sortable',
-        c: [0.70, 0.50],
+        c: [0.70 * 0.95, 0.50 * 0.95],
```

Comment:
this suite sets up `bidCpmAdjustment` by mocking `getGlobal`, which was not seen by `adjustBids` in `auction.js` because that looked directly at `$$PREBID_GLOBAL$$.bidderSettings`. This test never caught it, but there are other assertions in this suite that do expect numbers to get through the adjustment, so we need to modify either the setup or the assertion.